internal/cli/atlas/serverless/backup/snapshots/describe.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 MongoDB Inc
```

Comment:
[q] will we add E2E tests? (if it's captured in another JIRA ticket feel free to resolve)

Reply:
https://jira.mongodb.org/browse/CLOUDP-164770